src/scribe_data/cli/main.py

```diff
@@ -265,22 +272,34 @@ def main() -> None:
         help="Convert all languages and data types.",
     )
 
-    # MARK: Setup CLI
+    # MARK: Download
 
-    args = parser.parse_args()
-
-    if args.data_type and isinstance(args.data_type, str):
-        args.data_type = args.data_type.replace("-", "_")
+    download_parser = subparsers.add_parser(
+        "download",
+        aliases=["d"],
+        help="Download Wikidata dumps.",
+        description="Download Wikidata dumps from dumps.wikimedia.org.",
+        epilog=CLI_EPILOG,
+        formatter_class=lambda prog: argparse.HelpFormatter(prog, max_help_position=60),
+    )
+    download_parser._actions[0].help = "Show this help message and exit."
+    download_parser.add_argument(
+        "-wd",
+        "--wikidata-dump",
```

Comment:
```suggestion
        "--wikidata-dump-version",
```
I'm wondering if this should be something like the above to specify that this refers to the version of the dump to go download, i.e. either `latest` or a specific one given in the `YYYYMMDD` format.

Reply:
Accepted both suggestions, and I'm changing the short versions as well so they're more fitting :)